Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -901,6 +903,29 @@ def build_search_result(raw_response: dict, search_id: Union[str, int] = None):
         return f'{INTEGRATION_NAME} - Search did not find any result.', {}, {}
 
 
+def build_title_with_page_numbers(title: str, count: int, limit: int, offset: int) -> str:
```

Comment:
change `limit` to `page_size` in all the relevant places.

Reply:
switched in all commands except for search commands where page_size and limit are both used, and mean different things